vundo.el

```diff
@@ -291,6 +291,13 @@ after all the clean up the exiting function does. Ie, it is the
 very last thing that happens when vundo exists."
   :type 'hook)
 
+(defcustom vundo-diff-setup-hook nil
+  "List of functions to call after creating a diff buffer.
+This hook runs immediately after the ‘vundo-diff’ buffer is setup,
```

Comment:
Perhaps "This hook runs in the `vundo-diff' buffer immediately after it is set up", so the user knows they can do things in this buffer.

Reply:
Good idea.